apps/staking/app/mystakes/modules/NoticeModule.tsx

```diff
@@ -11,7 +13,11 @@ export default function NoticeModule() {
       <ModuleHeader>
         <ModuleText>{dict('title')}</ModuleText>
         <br />
-        {dict.rich('description')}
+        <br />
+        {dict.rich('description', {
+          link: externalLink(URL.DOCS),
+          'bridge-link': externalLink('https://getsession.org/'),
```

Comment:
Just want to confirmt that this is the correct link?

Reply:
its not, we'll be doing a full link refresh shortly